templates/web/base/admin/problem_row.html

```diff
@@ -37,6 +37,8 @@
         <td>[% prettify_state(problem.state, 1) %]<br><small>
             [% loc('Created') %]:&nbsp;[% PROCESS format_time time=problem.created %]
             <br>[% loc('When sent') %]:&nbsp;[% PROCESS format_time time=problem.whensent %]
+            [%- send_fail_bodies = problem.send_fail_bodies -%]
+            [%- IF send_fail_bodies %]<br>[% loc('Failed bodies:') %]&nbsp;[% send_fail_bodies.join(', ') %][% END -%]
```

Comment:
This needs a `.size` in the `IF` otherwise it is always true and displays a blank line.